callisto/delivery/models.py

```diff
@@ -157,7 +168,31 @@ def decrypted_report(self, key):
         Raises:
           CryptoError: If the key and saved salt fail to decrypt the record.
         """
-        return _decrypt_report(salt=self.salt, key=key, encrypted=self.encrypted)
+        iterations = None
+        harden_runtime = False
+        hasher = identify_hasher(self.encode_prefix)
+
+        if self.encode_prefix == '':
```

Comment:
we could, although it might be better instead to create a separate check function so we can make sure that on the empty string case we set the iterations and salt properly? (right now it anticipates a number of iterations changing that isn't the original case, and it'd need to return a 3rd value to indicate this so we can be sure to use the original iterations and salt from the model and not try to pull these from a nonexistent encode_prefix)